spec/bowlingSpec.js

```diff
@@ -0,0 +1,146 @@
+describe("Scorecard",() => {
+	
+	var scorecard;
+
+	beforeEach(() => {
+		scorecard = new Scorecard();
+		spyOn(console, 'log');
+	});
+
+	describe("getFrame", () => {
+
+		it("should return the current frame", () => {
+			let currentFrame = scorecard.getFrame();
+			expect(currentFrame).toEqual(0);
+		});
+
+	});
+
+	describe("getFramecard", () => {
+
+		it("should return framecard array", () => {
+			let framecardArr = scorecard.getFramecard();
+			expect(framecardArr).toEqual([]);
+		});
+
+	});
+
+	describe("inputRoll", () => {
+
+		it("will accept whole numbers between 0 and 10", () => { 
+			expect(scorecard.inputRoll(5)).toEqual (5);
+		});
+
+		it("will not accept numbers below 0", () => { 
+			scorecard.inputRoll(-2)
+			expect(console.log).toHaveBeenCalledWith("You must enter a number between 0 and 10");
+		});
+
+		it("will not accept numbers above 10", () => { 
+			scorecard.inputRoll(11)
+			expect(console.log).toHaveBeenCalledWith("You must enter a number between 0 and 10");
+		});
+
+	});
+
+	describe("addToFrame", () => {
+		
+		it("should add the roll to the current frame", () => {
+			let framecardArr = scorecard.getFramecard();
+			let currentFrame = scorecard.getFrame();
+			let roll1 = scorecard.inputRoll(5);
+			scorecard.addToFrame(roll1);
+			expect(framecardArr[currentFrame]).toContain (5);
+		});
+
+		it("should add two rolls to the current frame", () => {
+			let framecardArr = scorecard.getFramecard();
+			let currentFrame = scorecard.getFrame();
+			let roll1 = scorecard.inputRoll(5);
+			let roll2 = scorecard.inputRoll(5);
+			scorecard.addToFrame(roll1);
+			scorecard.addToFrame(roll2);
+			expect(framecardArr[currentFrame]).toEqual ([5,5]);
+		});
+
+		it("should fill out the frame if the first roll is 10", () => {
+			let framecardArr = scorecard.getFramecard();
+			let currentFrame = scorecard.getFrame();
+			let roll1 = scorecard.inputRoll(10);
+			scorecard.addToFrame(roll1);
+			expect(framecardArr[currentFrame]).toEqual ([10,0]);
+		});
```

Comment:
These tests are testing your specific implementation (that some numbers find themselves in a specific instance variable), rather than testing behaviour.